cfme/common/host_views.py

```diff
@@ -240,7 +241,7 @@ class HostsView(ComputeInfrastructureHostsView):
 
     @property
     def is_displayed(self):
-        return self.in_compute_infrastructure_hosts and self.title.text == "Hosts"
+        return self.in_compute_infrastructure_hosts and self.title.text in "Hosts / Nodes"
```

Comment:
I assume that `self.title.text` can be equal to `"Hosts"` or `"Nodes"` or `"Hosts / Nodes"`. So `self.title.text in "Hosts / Nodes"` handles all the cases.